sdk/highlight-go/highlight.go

```diff
@@ -255,6 +255,20 @@ func SetProjectID(id string) {
 	projectID = id
 }
 
+func SetLogOut(print bool) {
+	logOut = print
```

Comment:
I don't see why anyone would want this. If anything, this should be setting the min log level. For example, here's how logrus does it:
https://github.com/sirupsen/logrus#level-logging

Reply:
this came from a customer request to use highlight side by side with their existing logger that prints to console. i'll adjust this to customize a max level that logs will be emitted for.